### PR TITLE
Use search bar from commons-ui

### PR DESCRIPTION
**Pull request by klesaulnier**

### CODE REVIEW

**Review by EstherDarkish — APPROVED** [review]
Code OK Tests OK

**Review by jamal-khey — APPROVED** [review]
Test and Code OK
Console logs Ok

File: src/components/top-bar-equipment-seach-dialog/use-search-matching-equipments.tsx [hunk 1]
```diff
@@ -5,18 +5,16 @@
...
     getEquipmentsInfosForSearchBar,
-    useSnackMessage,
-    useDebounce,
+    useElementSearch,
 } from '@gridsuite/commons-ui';
-import { SEARCH_FETCH_TIMEOUT_MILLIS } from '../../utils/UIconstants';
 import { useNameOrId } from '../utils/equipmentInfosHandler';
 import { searchEquipmentsInfos } from '../../services/study';
 import { UUID } from 'crypto';
+import { Equipment } from '@gridsuite/commons-ui/dist/utils/EquipmentType';
```
Comment (sBouzols) [inline src/components/top-bar-equipment-seach-dialog/use-search-matching-equipments.tsx:17]: could be import directly from @gridsuite/commons-ui no ?

**Review by sBouzols — APPROVED** [review]
Code Review OK
Tests OK
Console warning check OK

**Pull request merged**